src/components/FirebaseFunctionsPageTable/FirebaseFunctionsPageTable.tsx

```diff
@@ -164,7 +164,7 @@ export const FirebaseFunctionsPageTable: React.FC = () => {
     <CircularProgress />
   ) : error ? (
     <Typography>
-      {'Error occured while loading available projects: ' + error}
+      {`Error occured while loading available projects: ${error}`}
```

Comment:
Does this look ok when an error shows up? Would it be slightly nicer to put the error in a `<pre/>` tag?

Reply:
Fixed, turned out that I don't even need to display an error here